### ISSUE TITLE
Enable streaming rendering

### ISSUE THREAD

**Issue by egil** [egil/Htmxor#16]

**Comment by tanczosm** [comment]
For streaming rendering just be careful because Htmx doesn't support streaming rendering at the moment.

**Comment by tanczosm** [comment]
Ah I forgot we discussed this exact thing.  Htmx natively just handles responses only when they have completed, so you end up having to wait out the entire streaming result   But yes, I did create an extension to handle that issue.  It will need some small updating for htmx 2.0 though since they changed the swapping api methods but the changes are just a few lines.  If you get streaming to work then it can work via an extension.

**Comment by tanczosm** [comment]
Here you go:

[https://github.com/JalexSocial/Rizzy/blob/main/src/Rizzy/wwwroot/js/blazor-streaming.js](https://github.com/JalexSocial/Rizzy/blob/main/src/Rizzy/wwwroot/js/blazor-streaming.js)

**Comment by egil** [comment]
Yeah, may current strategy is to completely remove blazer.web.js from the mix.
